src/node-red/core-nodes/nav.njk

```diff
@@ -0,0 +1,15 @@
+---
+pagination:
+    data: coreNodesArray
+    size: 1
+    alias: category
+    addAllPagesToCollections: true
+permalink: "node-red/core-nodes/{{ category.key | lower | replace(\" \", \"\") }}-nav/"
```

Comment:
I'm not sure what this page is, and why we're adding it?

Reply:
This is the page that generates the `(coreNodeCategory)-nav.njk`, one per category, so we can assign the eleventyNavigation.key that will be the parent of all the nodes from the same category.

We can't assign a parent if it doesn't corresponds with some file that has the same value as a key.

And since we don't have an index file for each category, we're redirecting to the first child of the group and making sure the empty page isn't generated, only the nav item is added in the navigation, I took the idea from https://www.11ty.dev/docs/plugins/navigation/#overriding-the-url even though it's not an external link.